fixtures/src/components/controlled/select.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { Select, Option } from 'carbon-react/lib/__experimental__/components/select';
+import { LogConsumer } from '../log';
+
+const ControlledSelect = () => {
+  const [selectedValue, setSelectedValue] = useState();
+  const [selectedValueExampleTwo, setSelectedValueExampleTwo] = useState('2');
```

Comment:
`useState('2')` -> `useState(['2'])` after Michael's Select PR gets merged

Reply:
I think it needs to be `useState('2')` and then `useState(['2'])` if the `enableMultiSelect` prop is true as this breaks the invariant otherwise

Reply:
Oh yeah, my bad, although I find it strange that in single select mode this component returns an array with just one value, but expects to be given a single value not wrapped in array.